src/output/icons.rs

```diff
@@ -8,111 +8,112 @@ struct Icons;
+    const LANG_JAVA: char =       '\u{e256}'; // 
+    const LANG_JAVASCRIPT: char = '\u{e74e}'; // 
+    const LANG_KOTLIN: char =     '\u{e634}'; // 
+    const LANG_OCAML: char =      '\u{e67a}'; // 
+    const LANG_PERL: char =       '\u{e67e}'; // 
+    const LANG_PHP: char =        '\u{e73d}'; // 
+    const LANG_PYTHON: char =     '\u{e606}'; // 
+    const LANG_R: char =          '\u{e68a}'; // 
+    const LANG_RUBY: char =       '\u{e21e}'; // 
+    const LANG_RUBYRAILS: char =  '\u{e73b}'; // 
+    const LANG_RUST: char =       '\u{e68b}'; // 
+    const LANG_SASS: char =       '\u{e603}'; // 
+    const LANG_STYLUS: char =     '\u{e600}'; // 
+    const LANG_TEX: char =        '\u{e69b}'; // 
+    const LANG_TYPESCRIPT: char = '\u{e628}'; // 
+    const LANG_V: char =          '\u{e6ac}'; // 
+    const LIBRARY: char =         '\u{eb9c}'; // 
+    const LICENSE: char =         '\u{f02d}'; // 
+    const LOCK: char =            '\u{f023}'; // 
+    const MAKE: char =            '\u{e673}'; // 
+    const MARKDOWN: char =        '\u{f48a}'; // 
+    const MUSTACHE: char =        '\u{e60f}'; // 
+    const NODEJS: char =          '\u{e718}'; // 
+    const NPM: char =             '\u{e71e}'; // 
+    const OS_ANDROID: char =      '\u{e70e}'; // 
+    const OS_APPLE: char =        '\u{f179}'; // 
+    const OS_LINUX: char =        '\u{f17c}'; // 
+    const OS_WINDOWS: char =      '\u{f17a}'; // 
+    const OS_WINDOWS_CMD: char =  '\u{ebc4}'; // 
+    const PLAYLIST: char =        '\u{f0cb9}'; // 󰲹
+    const POWERSHELL: char =      '\u{ebc7}'; // 
+    const PRIVATE_KEY: char =     '\u{f0306}'; // 󰌆
+    const PUBLIC_KEY: char =      '\u{f0dd6}'; // 󰷖
+    const RAZOR: char =           '\u{f1fa}'; // 
+    const REACT: char =           '\u{e7ba}'; // 
+    const README: char =          '\u{f00ba}'; // 󰂺
+    const SHEET: char =           '\u{f1c3}'; // 
+    const SHELL: char =           '\u{f1183}'; // 󱆃
+    const SHELL_CMD: char =       '\u{f489}'; // 
+    const SHIELD_CHECK: char =    '\u{f0565}'; // 󰕥
+    const SHIELD_KEY: char =      '\u{f0bc4}'; // 󰯄
+    const SHIELD_LOCK: char =     '\u{f099d}'; // 󰦝
+    const SIGNED_FILE: char =     '\u{f19c3}'; // 󱧃
+    const SLIDE: char =           '\u{f1c4}'; // 
+    const SUBLIME: char =         '\u{e7aa}'; // 
+    const SUBTITLE: char =        '\u{f0a16}'; // 󰨖
+    const TERRAFORM: char =       '\u{f1062}'; // 󱁢
+    const TEXT: char =            '\u{f15c}'; // 
+    const TYPST: char =           '\u{1D42D}'; // 𝐭
+    const UNITY: char =           '\u{e721}'; // 
+    const VECTOR: char =          '\u{f0559}'; // 󰕙
+    const VIDEO: char =           '\u{f03d}'; // 
+    const VIM: char =             '\u{e7c5}'; // 
+    const WRENCH: char =          '\u{f0ad}'; // 
+    const XML: char =             '\u{f05c0}'; // 󰗀
+    const YAML: char =            '\u{e6a8}'; // 
+    const YARN: char =            '\u{e6a7}'; // 
```

Comment:
please revert this

Reply:
Oh, I unformatted it incorrectly. Sorry for being so incompetent, I have little experience with contributing to big Rust projects, or working with big Rust projects in general.

Reply:
> Sorry for being so incompetent, I have little experience with contributing to big Rust projects, or working with big Rust projects in general.

Its okay I might have said it too rudely too